### PR TITLE
[#2874] Extend instructor help page

### LINKED ISSUE #2874: Instructor help page: mention the 'moderation' feature
also mention sending reminders individually

### PR DESCRIPTION
**Pull request by nodermatt**
Fixes #2874 

I added two new collapsible divs (based on the other examples) to document the the moderate and remind feature on the instructor help page. Since those features involve multiple steps there are no component mocks (as advised by @damithc ). Instead, the relevant button names were added to make the user aware of them. Section "How do I create and schedule a new feedback session?" does briefly mention the remind feature, but it doesn't appear in the search results while collapsed. Therefore adding it as a a separate question would be worth it IMHO.

### CODE REVIEW

File: src/web/app/pages-help/instructor-help-page/instructor-help-sessions-section/instructor-help-sessions-section.component.html [hunk 1]
```diff
@@ -678,7 +716,7 @@ <h4 *ngIf="displaySubsection(showQuestion, 10, 14)">Restoring Deleted Sessions</
     </div>
   </div>
   <!-- Question -->
-  <div *ngIf="showQuestion.length === 0 || showQuestion[12]" #question>
+  <div *ngIf="showQuestion.length === 0 || showQuestion[13]" #question>
```
Comment (madanalogy) [inline src/web/app/pages-help/instructor-help-page/instructor-help-sessions-section/instructor-help-sessions-section.component.html:719]: And `14` here

File: src/web/app/pages-help/instructor-help-page/instructor-help-sessions-section/instructor-help-sessions-section.component.html [hunk 2]
```diff
@@ -650,7 +688,7 @@ <h4 *ngIf="displaySubsection(showQuestion, 10, 14)">Restoring Deleted Sessions</
     </div>
   </div>
   <!-- Question -->
-  <div *ngIf="showQuestion.length === 0 || showQuestion[11]" #question id="restore-session">
+  <div *ngIf="showQuestion.length === 0 || showQuestion[12]" #question id="restore-session">
```
Comment (madanalogy) [inline src/web/app/pages-help/instructor-help-page/instructor-help-sessions-section/instructor-help-sessions-section.component.html:691]: Did you mean `13` here?

**Comment by wkurniawan07** [comment]
@nodermatt, there are some refactoring going on that changes the way this help page works. May I suggest to hold off this PR for a while, maybe 2-3 days.

File: src/web/app/pages-help/instructor-help-page/instructor-help-sessions-section/instructor-help-sessions-section.component.html (hunk 1 above)
Comment (nodermatt) [reply src/web/app/pages-help/instructor-help-page/instructor-help-sessions-section/instructor-help-sessions-section.component.html]: That's right. resolved.

File: src/web/app/pages-help/instructor-help-page/instructor-help-sessions-section/instructor-help-sessions-section.component.html (hunk 2 above)
Comment (nodermatt) [reply src/web/app/pages-help/instructor-help-page/instructor-help-sessions-section/instructor-help-sessions-section.component.html]: That's right. resolved.

**Comment by nodermatt** [comment]
Mea culpa, I didn't properly check the indices after adding the moderate feature block. Values are now updated and tested.

**Comment by wkurniawan07** [comment]
@nodermatt the refactoring are done and you can continue the work here. There will be some parts that need to be re-implemented.

**Comment by madanalogy** [comment]
Closing due to inactivity. If you'd like to continue working on this just reopen the PR.

**Pull request closed without merging**